### DISCUSSION TITLE
tooltip如何去react和antd

### DISCUSSION THREAD

**Discussion by xingwanying** [antvis/S2#247] [General]
目前core层tooltip组件依赖了antd和react，而umd目前react和antd都是external的方式，如果要用umd必须先引入react+antd。
方案一： toolip在core层归入interaction，渲染由外部自定义，内部不做实现
方案二：tooltip在core层原生简单实现，只需要有最基础的展示，也可以引入外部自定义组件    
or 其他方案

**Comment by qubaomingg** [comment]
赞成方案二，原生实现官网提供的极简tooltip

**Reply by hustcc** [reply]
参考这个 https://github.com/myliang/x-spreadsheet/blob/54f4be436fbc9b799bddb46859f2455b4a910b82/src/component/element.js

一个简单的文件，去构建一些相对简单的原生 html。
